Packs/VirusTotal/Integrations/VirusTotalV3/VirusTotalV3.py

```diff
@@ -14,6 +14,74 @@
 COMMAND_PREFIX = "vt"
 INTEGRATION_ENTRY_CONTEXT = "VirusTotal"
 
+INDICATOR_TYPE = {
```

Comment:
is this needed? can you achieve the same functionality  with `FeedIndicatorType`?

Reply:
In my opinion no, as FeedIndicatorType is only an object that contains the common types.
As we get the type from the api we need to convert the string to our convention of it which is done by this dict.
Do you think there is a better way?